karma.conf.js

```diff
@@ -72,17 +72,13 @@ module.exports = function (config) {
     logLevel: config.DEBUG,
 
     // jUnit Report output
-    reporters: ['dots'],
+    reporters: ['junit', 'dots'],
```

Comment:
@anotherchrisberry do we need the junit reporter or was this a bad idea to add?  I seem to remember something relying on it somehow?

Reply:
I think this used to give us a nice report in Jenkins, but that's not a great reason to keep it in, since we only run the build against Jenkins internally.

Not a big deal either way.